tests/accesscontrol/helper/helper.go

```diff
@@ -56,6 +56,22 @@ func DefineDeployment(replica int32, containers int, name string) (*appsv1.Deplo
 	return deploymentStruct, nil
 }
 
+func DefineDeploymentWithClusterRoleBindingwithServiceAccount(replica int32, containers int, name string) (*appsv1.Deployment, error) {
```

Comment:
Second "with" should be capitalized, as in DefineDeploymentWithClusterRoleBindingWithServiceAccount